apps/passport-client/components/shared/PCDArgs.tsx

```diff
@@ -56,20 +58,49 @@ export function PCDArgs<T extends PCDPackage>({
   options?: ArgsDisplayOptions<ArgsOf<T>>;
 }): JSX.Element {
   const [showAll, setShowAll] = useState(false);
+
+  // Flatten record arguments (if any), keeping track of the parent argument to
+  // properly mutate (cf. `setArg`) as well as inheriting argument fields from
+  // the record argument.  Validator parameters are also combined.
+  const flattenedArgs = Object.entries(args).flatMap(([argName, arg]) => {
```

Comment:
Add a type here for clarity, so a reader knows what flattenedArgs is meant to contain.